src/package/packageVersionCreate.ts

```diff
@@ -504,27 +504,39 @@ export class PackageVersionCreate {
       );
     }
 
-    // don't package the profiles from any un-packagedMetadata dir in the project
-    const profileExcludeDirs = this.project
-      .getPackageDirectories()
-      .map((packageDir) => (packageDir as PackageDescriptorJson).unpackagedMetadata?.path)
-      .filter((packageDirPath) => packageDirPath) as string[];
-
-    const typesArr =
-      this.options?.profileApi?.filterAndGenerateProfilesForManifest(packageXmlAsJson.types, profileExcludeDirs) ??
-      packageXmlAsJson.types;
-
-    // Next generate profiles and retrieve any profiles that were excluded because they had no matching nodes.
-    const excludedProfiles = this.options?.profileApi?.generateProfiles(
-      packageVersProfileFolder,
-      typesArr,
-      profileExcludeDirs
-    );
+    if (this.packageObject.scopeProfiles) {
+      this.logger.debug(
```

Comment:
I believe the below profile filtering also removes any unused settings from the "packaged" profiles (profiles in the scoped directory). We probably still want to keep this functionality as it could introduce new profile settings in subsequent package versions.

Reply:
Thanks!  I updated the code based on your comment.